tests/PHPStan/Parallel/SchedulerTest.php

```diff
@@ -87,6 +87,10 @@ public function testSchedule(
 		array $expectedJobSizes
 	): void
 	{
+		if ($numberOfFiles < 0) {
+			throw new \PHPStan\ShouldNotHappenException();
```

Comment:
I'd rather just have `0|positive-int` in the PHPDoc

Reply:
since phpstan does not verify the phpdoc against the phpunit-data-provider I figured it would be more safe to use a IF check here.

will change to your suggestion though